src/cdk-experimental/menu/menu.ts

```diff
@@ -259,6 +265,30 @@ export class CdkMenu extends CdkMenuGroup implements Menu, AfterContentInit, OnI
     }
   }
 
+  /**
+   * Subscribe to the menu trigger's open events in order to track the trigger which opened the menu
+   * and stop tracking it when the menu is closed.
+   */
+  private _subscribeToMenuOpen() {
```

Comment:
Not strictly necessary for this PR, but it would be good to cut down on the duplication between menu and menubar here. Maybe just add a TODO for now

Reply:
yeah, totally agree. Once everything is finalized I think pulling the duplicated logic out into an abstract Menu class is the way to go.